ci-operator/jobs/openshift/release/openshift-release-release-4.12-periodics.yaml

```diff
@@ -44,37 +44,42 @@ periodics:
 
         # if the source and destination are identical, do nothing
         export KUBECONFIG=/etc/app.ci/kubeconfig
-        from=$( oc get istag -n "${TAG_NS}" "${TAG_SOURCE}" -o template='{{ .image.metadata.name }}' )
+        from=$( oc get istag -n "${TAG_NS}" "${TAG}:${BRANCH}" -o template='{{ .image.metadata.name }}' )
         to=$( oc get istag --ignore-not-found -n ocp "${BRANCH}:${TAG}" -o template='{{ .image.metadata.name }}' )
+        from_extensions=$( oc get istag -n "${TAG_NS}" "${TAG}-extensions:${BRANCH}" -o template='{{ .image.metadata.name }}' )
+        to_extensions=$( oc get istag --ignore-not-found -n ocp "${BRANCH}:${TAG}-extensions" -o template='{{ .image.metadata.name }}' )
         unset KUBECONFIG
         if [[ "${from}" == "${to}" ]]; then
-          echo "info: ${TAG_NS}/${TAG_SOURCE} matches ocp/${BRANCH}:${TAG}"
+          echo "info: ${TAG_NS}/${TAG}:${BRANCH} matches ocp/${BRANCH}:${TAG}"
           exit 0
         fi
-        echo "Will promote ${from}, current is ${to}"
+        echo "Will promote ${TAG} ${from}, current is ${to}"
+        echo "Will promote ${TAG}-extensions ${from_extensions}, current is ${to_extensions}"
 
         # error out if the image isn't on quay
         to_quay="quay.io/openshift-release-dev/ocp-v4.0-art-dev@${from}"
-        if ! oc image info -a "/usr/local/pull-secret/.dockerconfigjson" "${to_quay}"; then
-          echo "error: The source image has not been pushed to quay ${from}"
-          exit 1
-        fi
+        to_quay_extensions="quay.io/openshift-release-dev/ocp-v4.0-art-dev@${from_extensions}"
+        for img in ${to_quay} ${to_quay_extensions}; do 
```

Comment:
@cgwalters There is an extra space at the end of this line that is causing the issue with `make jobs`.  Removing it and re-running the command should resolve the problem.

Reply:
Oh wow.  The chance I would have figured that out on my own is basically 0%.  I assume you got burned by this in the past or something?